docs/collecting-data/collecting-data-from-third-parties/adjust-webhook/index.md

```diff
@@ -29,7 +29,7 @@ The guide is illustrative. One of the nice things about Adjust is that it gives
 
 Log into Adjust and navigate to Settings -> Events screen in the dashboard. Hover over the install event, click the “edit” icon that appears and enter the following, substituting your own collector URL for `mycollector.mydomain.com`:
 
-```
+```markup
```

Comment:
Not on the supported languages list... why `markup`?

Reply:
Its in the first language on https://prismjs.com/#supported-languages - it nicely colours it so that the parameters are separated out.  The uri/url option was quite dull in comparison.

Reply:
I am blind 🤦‍♂️